src/lib/calculation.ts

```diff
@@ -1,7 +1,11 @@
-import floor from 'lodash.floor'
 import { ChildInput, TaxFormUserInput } from '../types/TaxFormUserInput'
 import { Child, TaxForm } from '../types/TaxForm'
-import { getRodneCisloAgeAtYearAndMonth, parseInputNumber } from './utils'
+import {
+  getRodneCisloAgeAtYearAndMonth,
+  floorDecimal,
+  parseInputNumber,
+} from './utils'
+import Decimal from 'decimal.js'
 
```

Comment:
Metoda `round2decimal` by sa uz nemala po spravnosti nikde pouzivat, ked mame decimal libku, nie? je to riadok 15, ale nemozem tam komentovat :D

Reply:
Hej, nemala by byt uz pouzivana. Vymazem ju.